### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ezpzbz**
This PR:
* Moves the build system to `flit`
* Has some bug fixes and improvements in examples. 
* Has updated `pre-commit` stuffs in line with `aiida-core`

### CODE REVIEW

File: examples/example_0.py [hunk 1]
```diff
@@ -72,7 +73,7 @@ def cli(codelabel, submit):
     try:
         code = Code.get_from_string(codelabel)
     except NotExistent:
-        print("The code '{}' does not exist".format(codelabel))
+        print('The code {codelabel} does not exist.')
```
Comment (danielhollas) [inline examples/example_0.py:76]: I think this is missing the `f` at the beginning to make it an f-string.

File: examples/example_3.py [hunk 2]
```diff
@@ -79,7 +80,7 @@ def cli(codelabel, previous_calc, submit):
     try:
         code = Code.get_from_string(codelabel)
     except NotExistent:
-        print("The code '{}' does not exist".format(codelabel))
+        print('The code {codelabel} does not exist.')
```
Comment (danielhollas) [inline examples/example_3.py:83]: Same here and in example 4 and 5.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    "aiida_core[atomic_tools] >= 1.6.0, <2.0.0",
+    "periodictable"
+]
+
+[project.urls]
+Home = "https://github.com/pzarabadip/aiida-orca"
+
+[project.entry-points."aiida.calculations"]
+"orca.orca" = "aiida_orca.calculations:OrcaCalculation"
+"orca.ase" = "aiida_orca.calculations:OrcaAsaCalculation"
```
Comment (danielhollas) [inline pyproject.toml:36]: Should this be `orca.asa`?

**Review by danielhollas — COMMENTED** [review]
@pzarabadip nice! :rocket: 

Had a quick look and spotted one possible issue in pyproject.toml

**Comment by ezpzbz** [comment]
Thanks for these. I just pushed fixes.

**Pull request merged**